server/src-lib/Hasura/RQL/DML/Mutation.hs

```diff
@@ -212,3 +214,6 @@ mkSelCTEFromColVals qt allCols colVals =
       TENull          -> S.SENull
       TELit textValue ->
         S.withTyAnn (unsafePGColumnToRepresentation colTy) $ S.SELit textValue
+      TEJSON (Q.JSON j) -> 
+        S.withTyAnn (unsafePGColumnToRepresentation colTy) $ S.SELit $ jsonText
```

Comment:
Is getting a `stringified json` here okay?

I assumed that since we are just logging the SQL query here. Having a json as string doesn't harm. If not we would have to add a new constructor for `S.SELit` which takes json value